### ISSUE TITLE
Settings screen text contrast is not accessible

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#25790]
### Steps to reproduce

1. Open settings
2. Be concerned there's a 5% tint on everything

#### What did you expect?

Higher contrast, I think. The settings overall feel unnervingly dark/slightly too hard to read. 

#### What happened instead?

![image](https://github.com/vector-im/element-web/assets/1190097/3963324f-ea1d-4983-8751-a862c36641d0)

### Operating system

Windows 11

### Application version

Element Nightly version: 0.0.1-nightly.2023071301 Olm version: 3.2.14

### How did you install the app?

The Internet

### Homeserver

t2l.io

### Will you send logs?

No

**Comment by germain-gg** [comment]
I'm slightly unsure what the screenshot above check? 

The background color on the modal is `--cpd-bg-canvas-default` which corresponds to `#101317`. What are the two HEX codes in above?

And pretty much all the text in the screenshot that Travis posted in using `--cpd-color-text-secondary` which corresponds to `#808994`

The combination of the two colour above yield a `5.25:1` contrast ratio, passing `AA` and failing `AAA`

**Comment by t3chguy** [comment]
> What are the two HEX codes in above?

They're out of `Digital Colour Meter.app` so must be taken with a slight grain of salt. Found that to be more reliable than trying to chase down the underlying hex as we sometimes use opacity.

Using FF can confirm the 5.25 ratio

![image](https://github.com/vector-im/element-web/assets/2403652/1440b7f0-f4d0-4737-9138-6441d504d111)

**Comment by germain-gg** [comment]
Closing, please feel free to re-open if we can spot places in settings that do not meeting the WCAG AA requirements
